brands/amenity/bank.json

```diff
@@ -4701,6 +4701,15 @@
       "name": "State Bank of India"
     }
   },
+  "amenity/bank|State Employees Credit Union": {
+    "tags": {
+      "amenity": "bank",
+      "brand": "State Employees Credit Union",
+      "brand:wikidata": "Q7603196",
+      "brand:wikipedia": "en:State Employees Credit Union",
+      "name": "State Employees Credit Union "
```

Comment:
```suggestion
      "name": "State Employees Credit Union",
      "short_name": "SECU"
```

Reply:
Hi, I creating a new branch and added your suggested changes. My apologizes for not doing this the first time. This is my first contribution.